ztp/gitops-subscriptions/argocd/PowerConfiguration.md

```diff
@@ -0,0 +1,224 @@
+# Power Configuration
+
+Low latency, high performance edge deployments require C-states and P-states to be disabled. This essentially results in each cpu running at the same maximum turbo frequency that is supported for all processors on that particular server. This provides the best latency for workloads, but also consumes the most power. However, not all workloads require such power intensive configurations. Workloads can be categorized as either *critical* or *non-critical*. Critical workloads require disabled C-state and P-state settings for high performance and low latency, whereas non-critical workloads use C-state and P-state settings for power savings at the cost of some latency /performance for those non-critical workloads.
```

Comment:
Change "disabled" to "disabled or limited". Change "This essentially results" to "This may result". Change "Critical workloads require disabled" to "Critical workloads require disabled or limited".